test/MUXControlsTestApp/App.xaml

```diff
@@ -35,6 +35,153 @@
+                                                <Setter Target="MyIcon.IconSource">
+                                                    <Setter.Value>
+                                                        <FontIconSource FontFamily="Segoe MDL2 Assets" Glyph="&#xE946;"/>
+                                                    </Setter.Value>
+                                                </Setter>
+                                            </VisualState.Setters>
+                                        </VisualState>
+                                        <VisualState x:Name="Success">
+                                            <VisualState.Setters>
+                                                <Setter Target="MyBrush.Color" Value="#107C10"/>
+                                                <Setter Target="MyIcon.IconSource">
+                                                    <Setter.Value>
+                                                        <FontIconSource FontFamily="Segoe MDL2 Assets" Glyph="&#xF13E;"/>
+                                                    </Setter.Value>
+                                                </Setter>
+                                            </VisualState.Setters>
+                                        </VisualState>
+                                        <VisualState x:Name="Default">
+                                            <VisualState.Setters>
+                                                <Setter Target="MyBrush.Color" Value="#69797E"/>
+                                            </VisualState.Setters>
+                                        </VisualState>
+                                        <VisualState x:Name="None">
+                                            <VisualState.Setters>
+                                                <Setter Target="MyBrush.Color" Value="Transparent"/>
+                                            </VisualState.Setters>
+                                        </VisualState>
+                                    </VisualStateGroup>
+                                </VisualStateManager.VisualStateGroups>
+
+                                <Grid x:Name="ContentRootGrid" >
+
+                                    <Grid.ColumnDefinitions>
+                                        <ColumnDefinition Width="1*"/>
+                                        <ColumnDefinition Width="1*"/>
+                                        <ColumnDefinition Width="2*"/>
+                                        <ColumnDefinition Width="7*"/>
+                                        <ColumnDefinition Width="1*"/>
+                                        <ColumnDefinition Width="1*"/>
+                                    </Grid.ColumnDefinitions>
+
+                                    <Rectangle x:Name="MyStatusColor"  Height="70" Width="50" Grid.Column="0" HorizontalAlignment="Left">
```

Comment:
nit: We could drop the "My" and just use StatusColor as the name.. same for below.

Reply:
Sure! I can do that for title, message, etc..., as well.